### DISCUSSION TITLE
gongju/fastjson

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#171] [Announcements]
# gongju/fastjson

01、前世今生 我是 fastjson，是个地地道道的杭州土著，但我始终怀揣着一颗走向全世界的雄心。这不，我在 GitHub 上的简介都换成了英文，国际范十足吧？

https://tobebetterjavaer.com/gongju/fastjson.html

**Comment by 999bug** [comment]
垃圾的框架，各种莫名其妙的bug。
